docs/uninstall-olm.md

```diff
@@ -0,0 +1,53 @@
+# How do I uninstall OLM?
+
+You can uninstall OLM in a symmetrical way as you did to install it. 
```

Comment:
This is not very clear. I would simply highlight that OLM can be uninstalled by deleting the Kubernetes resources that were added to the cluster when it was installed.

Reply:
+1 on this one. 

Assume that the person who installed olm is not the person uninstalling it. That might help with the rest of the doc too, coz you'd have to explain an uninitiated user everything from scratch, which is what this doc needs.